packages/jest.config.js

```diff
@@ -0,0 +1,201 @@
+/*
```

Comment:
@takeutak Is this file scoped for the entire project (e.g. all packages) or just a sub-set of packages?

cc: @sfuji822 @hartm @jonathan-m-hamilton

Reply:
Thank you for your suggestion.
As I said on the maintainer call, this file should be included by PR #444, not by this PR.  Then I deleted this file.
Moreover, I answered your question on the comment on PR #444.